### PR TITLE
Release v1.8.0

### PR DESCRIPTION
**Pull request by ifyouseewendy**
### WHY are these changes introduced?

Bump version to v1.8.0 as a feature release for scripts update, which includes radical changes about how a script is compiled. More details in https://github.com/Shopify/script-service/issues/2424

### WHAT is this pull request doing?

Bump version to v1.8.0

### Update checklist

- [x] I've added a CHANGELOG entry for this PR (if the change is public-facing)
- [x] I've considered possible cross-platform impacts (Mac, Linux, Windows).

### CODE REVIEW

**Comment by ifyouseewendy** [comment]
CI is failing due to a breaking change for shopify robocop.yml cannot be found. It's tracked [here](https://shopify.slack.com/archives/C01MXHNTT4Z/p1616513917418800)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Unreleased
 ------
 
+Version 1.8.0
+-------------
+
+* [1119](https://github.com/Shopify/shopify-app-cli/pull/1129): Update EP version and enable guest serialization for scripts
```
Comment (mllemango) [inline CHANGELOG.md:7]: Is this the right pr #? 😅

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 Unreleased
 ------
 
+Version 1.8.0
+-------------
```
Comment (andyw8) [inline CHANGELOG.md:5]: The PR link is incorrrect.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,11 @@
...
+
+* [1119](https://github.com/Shopify/shopify-app-cli/pull/1119): Update EP version and enable guest serialization for scripts
```
Comment (mllemango) [inline CHANGELOG.md:7]: I'm not sure I fully understand what this changelog entry means, could it be simplified to `enable guest serialization for scripts`?

**Comment by ifyouseewendy** [comment]
Just reword the change log to be `Enable guest serialization for scripts`.

I pushed another commit to update rubocop-shopify. As CI is failing ([build](https://travis-ci.com/github/Shopify/shopify-app-cli/jobs/493176062)) for

```
Running RuboCop...

404 "Not Found" while downloading remote config file http://shopify.github.io/ruby-style-guide/rubocop.yml
```

According to the [#ruby-style-guide](https://shopify.slack.com/archives/C0DRQ0HEJ/p1616521204019400?thread_ts=1616517819.006800&cid=C0DRQ0HEJ), upgrading rubocop-shopify should fix
the issue.

File: CHANGELOG.md (hunk 3 above)
Comment (andyw8) [reply CHANGELOG.md]: Also, consider if we need to provide further context here. Think of it from the perspective of a third-party developer upgrading from an older shopify_app, who is reading the changelog to understand if they need to change anything in their app.

**Review by mllemango — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (ifyouseewendy) [reply CHANGELOG.md]: @andyw8 Sorry, just noticed your comment from my email notification. So far, script is still mostly used internally. We simply want a version bump which can be pointed to for the script users. But yeah, your comments make sense. I'll keep that in mind for the future release.

**Pull request merged**